src/rpc/misc.cpp

```diff
@@ -174,6 +174,14 @@ UniValue validateaddress(const JSONRPCRequest& request)
             "  \"ismine\" : true|false,        (boolean) If the address is yours or not\n"
             "  \"iswatchonly\" : true|false,   (boolean) If the address is watchonly\n"
             "  \"isscript\" : true|false,      (boolean) If the key is a script\n"
+            "  \"script\" : \"type\"           (string, optional) The output script type. Possible types: nonstandard, pubkey, pubkeyhash, scripthash, multisig, nulldata, witness_v0_keyhash, witness_v0_scripthash\n"
```

Comment:
This is how it looks. Mind the `\"`.
<img width="750" alt="bildschirmfoto 2017-06-29 um 20 57 24" src="https://user-images.githubusercontent.com/178464/27705229-917574e0-5d0d-11e7-8950-cf93b8a9166d.png">

Sorry for the bike-shed :)